packages/wallet/backend/migrations/20230728101910_update_transactions_columns.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.alterTable('transactions', async (table) => {
+    table.timestamp('expiresAt')
+    await knex.raw(
+      [
+        `ALTER TABLE "transactions" DROP CONSTRAINT IF EXISTS "transactions_status_check";`,
+        `ALTER TABLE "transactions" ADD CONSTRAINT "transactions_status_check" CHECK ("status" = ANY (ARRAY['PENDING'::text, 'COMPLETED'::text, 'REJECTED'::text, 'EXPIRED'::text]))`
+      ].join('\n')
+    )
```

Comment:
`knex` does not have support to alter `enums` since they are only some check constraints.